src/fontra/client/web-components/reference-font.js

```diff
@@ -183,4 +194,142 @@ function readFileAsync(file) {
   });
 }
 
+async function getOPFSInfo() {
+  try {
+    return navigator.storage.estimate().then((info) => {
+      return {
+        free: info.quota - info.usage,
+        usage: info.usage,
+        percent: Math.round((info.usage / info.quota) * 100),
+        quota: info.quota,
+      };
+    });
+  } catch (error) {
+    dialog(
```

Comment:
Is this actually happening? Why the need for a dialog? Sounds like if this fails we have bigger problems to worry about, and I'm not sure the end user is helped in any way by this message.

Reply:
This happens only if the browser doesn't support `navigator.storage`, it is nicely supported but there are some exceptions like IE and Safari < 15.2: 
https://caniuse.com/?search=navigator.storage

Showing the dialog may be not useful to the user, but it could be for us. 
In case a user reports us a problem including an error message it would be easier for us to understand what is happening, in all other cases it's just a dead dialog.

Reply:
We don't support IE at all, and we also do not support Safari < 15.4 (we use `array.at()`, which is Safari >= 15.4)

I don't like debug dialogs in arbitrary places, because there is literally no good rationaly for when to do or not do it. We need to see console output in case of problems anyway. Later we could perhaps do a more proper up-front browser compatibility check, before we load anything important.